doc/_static/css/custom.css

```diff
@@ -98,6 +98,20 @@ code.descclassname {
 .highlight {
     background: rgba(255, 165, 0, 0.1);
 }
+<<<<<<< HEAD
```

Comment:
This looks like something left from a merge conflict.

Reply:
removed